examples/src/main/resources/hello_world_config.yml

```diff
@@ -1,6 +1,8 @@
 Grpc:
     server.port: 50051
     server.executorThreads : 4
+    grpcFilterConfig:
```

Comment:
Access log can be a top level config.

Reply:
If there are other config params other than accesslog, then this filterconfig can be used.
did not want to keep accesslog related config separate